test/InterchainTokenServiceUpgradeFlow.js

```diff
@@ -19,7 +19,7 @@ const Create3Deployer = getContractJSON('Create3Deployer');
 const MINT_BURN = 4;
 
 describe('Interchain Token Service Upgrade Flow', () => {
-    let wallet, otherWallet, signer;
+    let wallet, otherWallet, multisig;
```

Comment:
```suggestion
    let wallet, otherWallet, operator;
```
for clarity of the role. it's not a multisig, and the contract doesn't care about it

Reply:
I see. updated.